### DISCUSSION TITLE
Can i get a clean build?

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#14384] [General]
https://cirrus-ci.com/task/6221670078742528 i m facing this problem in LineageOS and only solution is clean build, trees are not the problem as i m using same trees to make PixelOS and i m able to compile it successfully https://cirrus-ci.com/build/5992824171659264 
link to trees https://github.com/Alone0316/android_device_samsung_a52sxq/tree/pixel
@Apon77

**Comment by Apon77** [comment]
1. It is 1 month before log! Which does not says anything current situation 
2. Your codes are not buildable. @Alone0316
![Screenshot_20221014-095347_Termux](https://user-images.githubusercontent.com/61594531/195759413-77f3ff9e-5375-4443-8b52-23761abc2d9e.png)
![Screenshot_20221014-095453_Plus](https://user-images.githubusercontent.com/61594531/195759483-a9365ebc-60a0-4455-815f-e9df9916f27c.png)
3. Even today had los 19.1 succeed build. So no clean is needed i think.

**Reply by Alone0316** [reply]
> 1. It is 1 month before log! Which does not says anything current situation
> 2. Your codes are not buildable. @Alone0316
>    ![Screenshot_20221014-095347_Termux](https://user-images.githubusercontent.com/61594531/195759413-77f3ff9e-5375-4443-8b52-23761abc2d9e.png)
>    ![Screenshot_20221014-095453_Plus](https://user-images.githubusercontent.com/61594531/195759483-a9365ebc-60a0-4455-815f-e9df9916f27c.png)
> 3. Even today had los 19.1 succeed build. So no clean is needed i think.

1. I will make again and give you latest logd
2. My codes are buildable but as LineageOS was having problems so i m making PixelOS and my local manifest is adapted fo PixelOS and as you know Lineage's repo naming scheme contain android_* before every repo that's why you are getting sync failure @Apon77
